app/models/notification.rb

```diff
@@ -5,13 +5,19 @@ class Notification < ApplicationRecord
   scope :recent_unread, -> { unread.order(created_at: :desc).limit(UNREAD_LIMIT) }
   scope :unread, -> { where(read_at: nil) }
 
-  def self.create_for(users, args = {})
+  # def self.create_for(users, args = {})
```

Comment:
I'm curious about this chunk being commented out.  Are you intending to bring this back?